<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LameCodecWidget</class>
 <widget class="QWidget" name="LameCodecWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,1">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Preset:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="presetComboBox">
       <property name="toolTip">
        <string>Either use one of lames's presets or your own settings.</string>
       </property>
       <item>
        <property name="text">
         <string>Medium</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Standard</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Extreme</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Insane</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Specify bitrate</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>User defined</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="presetBitrateSpinBox">
       <property name="suffix">
        <string> kbps</string>
       </property>
       <property name="minimum">
        <number>8</number>
       </property>
       <property name="maximum">
        <number>320</number>
       </property>
       <property name="value">
        <number>192</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="presetBitrateCbrCheckBox">
       <property name="toolTip">
        <string>Encode using a constant bitrate.
Only works with 80, 96, 112, 128, 160, 192, 224, 256 and 320 kbps</string>
       </property>
       <property name="text">
        <string>CBR</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>8</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="presetFastCheckBox">
       <property name="toolTip">
        <string>Use a faster encoding algorithm (results in a slightly lower output quality).</string>
       </property>
       <property name="text">
        <string>Fast encoding</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="presetStackedWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Mode:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="modeComboBox">
         <item>
          <property name="text">
           <string>Quality</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Bitrate</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="qualitySlider">
         <property name="minimum">
          <number>8</number>
         </property>
         <property name="maximum">
          <number>320</number>
         </property>
         <property name="pageStep">
          <number>8</number>
         </property>
         <property name="sliderPosition">
          <number>192</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="qualitySpinBox">
         <property name="minimum">
          <number>8</number>
         </property>
         <property name="maximum">
          <number>320</number>
         </property>
         <property name="value">
          <number>192</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>8</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Bitrate mode:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="bitrateModeComboBox">
         <item>
          <property name="text">
           <string>Variable</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Average</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Constant</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="cmdArgumentsCheckBox">
       <property name="text">
        <string>Additional encoder arguments:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="cmdArgumentsLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="manPageButton">
       <property name="toolTip">
        <string>Show man page</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="dialog-information"/>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>252</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
